src/utils/Processing.ts

```diff
@@ -203,19 +213,43 @@ export function loadExternalDependencies(
           `    uri: ${dep.uri ?? 'http://my-fhir-ig.org/ImplementationGuide/123'}\n` +
           '    version: current'
       );
-      continue;
+      return Promise.resolve();
+    } else if (EXT_PKG_TO_FHIR_PKG_MAP[dep.packageId]) {
+      // It is a special "virtual" FHIR extensions package indicating we need to load supplemental
+      // FHIR versions to support "implied extensions".
+      if (dep.version !== fhirVersion) {
+        logger.warn(
+          `Incorrect package version: ${dep.packageId}#${dep.version}. FHIR extensions packages ` +
+            "should use the same version as the implementation guide's fhirVersion. Version " +
+            `${fhirVersion} will be used instead. Update the dependency version in ` +
+            'sushi-config.yaml to eliminate this warning.'
+        );
+      }
+      logger.info(
+        `Loading supplemental version of FHIR to support virtual extensions package: ${dep.packageId}`
+      );
+      return loadSupplementalFHIRPackage(EXT_PKG_TO_FHIR_PKG_MAP[dep.packageId], defs);
+    } else {
+      return loadDependency(dep.packageId, dep.version, defs).catch(e => {
+        logger.error(`Failed to load ${dep.packageId}#${dep.version}: ${e.message}`);
+      });
     }
-    dependencyDefs.push(
-      loadDependency(dep.packageId, dep.version, defs)
-        .then(def => {
-          return def;
-        })
-        .catch(e => {
-          logger.error(`Failed to load ${dep.packageId}#${dep.version}: ${e.message}`);
-        })
-    );
-  }
-  return dependencyDefs;
+  });
+
+  return Promise.all(promises).then(() => {});
+}
+
+export async function loadSupplementalFHIRPackage(
```

Comment:
Should this go in `load.ts`? I thought `loadExternalDependencies` only gets its special spot in `Processing.ts` because it is used directly in `app.ts`.

Reply:
I think I put it here originally because it was initially a `private` function (used only by `loadExternalDependencies`).  But then I had to make it `public` in order to unit test it independently.

I can move it.

Reply:
Fixed in e4448ef